src/lib.rs

```diff
@@ -4,13 +4,13 @@ use std::error::Error;
 use std::path::PathBuf;
 
 mod checks;
-mod common;
+pub mod common;
```

Comment:
I think it would be better to use `pub(crate)` in this place.

Reply:
@DDtKey good catch, however this code has been removed